setuptools/command/egg_info.py

```diff
@@ -475,8 +475,7 @@ def global_exclude(self, pattern):
         return self._remove_files(match.match)
 
     def append(self, item) -> None:
-        if item.endswith('\r'):  # Fix older sdists built on Windows
-            item = item[:-1]
+        item = item.removesuffix('\r')
```

Comment:
Might wanna keep this comment, I'm guessing it was removed by an unsafe autofix
```suggestion
        item = item.removesuffix('\r')  # Fix older sdists built on Windows
```